build.gradle

```diff
@@ -40,6 +40,17 @@ allprojects {
                 url = 'https://repository.apache.org/content/repositories/snapshots'
             }
         }
+        println "Check 1:${System.getenv('GITHUB_ACTOR')?.size()} 2:${System.getenv('GH_TOKEN')?.size()} 3:${prj.name == 'grails-plugin-gsp'?:'skip'} 4:${!sitemeshVersion.endsWith('-SNAPSHOT')} 5:${System.getenv('DEVELOCITY_BUILD_CACHE_NODE_USER')?.size()}"
+        if (System.getenv("GITHUB_ACTOR") && System.getenv("GH_TOKEN")
```

Comment:
Why do you need to check for `GITHUB_ACTOR` && `GH_TOKEN`?

Reply:
It rights a username & Password.